soc/nxp/mcx/mcxw/Kconfig.defconfig

```diff
@@ -12,4 +12,23 @@ config SYS_CLOCK_HW_CYCLES_PER_SEC
 config MCUX_FLASH_K4_API
 	default y
 
+if BT
+
+# Include intercore messaging component
+config NXP_RF_IMU
+	default y
+
+#  Set the controller's public identity using NXP vendor command
+config BT_HCI_SET_PUBLIC_ADDR
+	default y
+
+# HCI RX buffers are received in ISR context. RX messages
+# need to be queued and processed by a dedicated thread
+config HCI_NXP_RX_THREAD
+	default y
```

Comment:
I guess on other SOC they aren't received in ISR context ? is the hardware architecture different ?

Reply:
Yes, RW and MCXW7x platforms have some HW differences and we use different drivers to drive the intercore communication. On RW, the user callback is called from a task, so no need for a dedicated thread at zephyr level. But, on MCXW7x, the user callback is called from ISR context, so we need a task at zephyr level (otherwise you get an assert when sending the packet to the ble host). This is something we will improve at driver level for MCXW7x for the next SDK releases btw, but it will take time to be implemented and validated so for now we have to do with what we have.

Reply:
yes , not expected to be addressed in this PR